cg/models/demultiplex/run_parameters.py

```diff
@@ -89,15 +101,75 @@ def __repr__(self):
         )
 
 
+class RunParametersHiSeq(RunParameters):
```

Comment:
is this class only for the 2500 or for the X as well? Two difference machines, I assume different run parameters as well

Reply:
They are actually very similar. The fields (or XML nodes) of interest, which are the number of read and index cycles, are structured identically. Other fields with information not relevant to the sample sheet construction may vary. Thus, the unified class for both flow cells